Apps/WebClient/src/ClientApp/app/components/timeline/laboratory.vue

```diff
@@ -242,5 +247,9 @@ export default class LaboratoryTimelineComponent extends Vue {
         this.isLoadingDocument = false;
       });
   }
+
+  private get reportAvailiable(): boolean {
```

Comment:
Redundant for flexibility. Will be removed once we get confirmation that the flag is pass uniquely.